src/cards/heal_mass.ts

```diff
@@ -8,7 +8,7 @@ import { CardRarity, probabilityMap } from '../types/commonTypes';
 import { heal_greater_id } from './heal_greater';
 
 export const heal_mass_id = 'Mass Heal';
-const healAmount = 10;
+const healAmount = 40;
```

Comment:
So regular heal gives 2hp to 1 mana ratio and this does 1hp to 1mana ratio, but it heals all allies.  I agree that the heal amount should be increased, 10 is pretty weak but maybe it should cost a little more?

Reply:
I did also increase the cost to 40. If you mean more than that... maybe?

Given it's a 3rd tier upgrade forbidden rarity spell, I don't mind making it a little more powerful than not